### DISCUSSION TITLE
How to commit files manually placed

### DISCUSSION THREAD

**Discussion by DanielFarmer** [community/community#47018] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, 

I have overwritten some files in a space normally controlled via github. 

I did this simply by using the file system to replace.

Is it possible to now get Github to recognize those new files?   

like a manual override? ( or a manual commit ) ?

**Comment by Hangell** [comment]
## Para que o GitHub reconheça os arquivos substituídos, você precisa fazer um "commit"

1. Abra o terminal na pasta do seu repositório local do GitHub
2. Adicione os arquivos substituídos ao "index" do Git usando o seguinte comando:

~~~shell
git add .
~~~

3. Faça o "commit" dos arquivos substituídos com uma mensagem descrevendo a alteração, usando o seguinte comando:

~~~shell
git commit -m "Sua mensagem de commit aqui"
~~~

4. Envie os arquivos substituídos para o repositório remoto no GitHub usando o seguinte comando:

~~~shell
git push origin nome-da-sua-branch
~~~

>Faz o backup em outra Branch antes ou ate mesmo local, depois busca git flow é uma forma de trabalhar com repositorios muito util, você vai utilizar uma branch como principal que é a de produção (master ou main), a branch (dev ou developer) como homolocação e as demais como feature, hotfix e afins para mergear com a dev e depois de homologar mesclar com a master isso evita esses tipos de problemas

Nest [link](https://medium.com/@diegowribeiro/trabalhando-com-git-e-git-flow-no-dia-a-dia-b%C3%A1sico-96a3ae02f8e3) tem algumas orientações de como utilizar o git flow
